test/lib/absinthe/phase/document/arguments/normalize_test.exs

```diff
@@ -43,6 +55,19 @@ defmodule Absinthe.Phase.Document.Arguments.NormalizeTest do
     }
   """
 
+  @fragment_query """
+    query Things($id: ID!) {
+      things {
+        ... thingsFragment
+      }
+    }
+    fragment thingsFragment on Things {
+      items(id: $id) {
+        id
+      }
+    }
+  """
```

Comment:
So without the fix this test fails:

```
 1) test when providing an input to a fragment normalizes the input (Absinthe.Phase.Document.Arguments.NormalizeTest)
     test/lib/absinthe/phase/document/arguments/normalize_test.exs:96
     Assertion with == failed
     code: %Blueprint.Input.String{value: "baz"} == id_argument.input_value().normalized()
     lhs:  %Absinthe.Blueprint.Input.String{errors: [], flags: %{},
            schema_node: nil, source_location: nil, value: "baz"}
     rhs:  nil
     stacktrace:
       test/lib/absinthe/phase/document/arguments/normalize_test.exs:101: (test)
```

Essentially I wanted to ensure that the arguments are correctly normalized in to the fragment node.

The ultimate behavior is that during query resolution for the above query, the arguments map passed to the resolver is empty. I can write a higher level test for that as well to help illustrate the issue if you want. (If so, any pointers to where that test should live?)